tests/KdybyTests/CsobPaymentGateway/HttpClientMock.php

```diff
@@ -71,9 +71,15 @@ private function resolveTargetFile($method, $url, $headers, $body)
 	{
 		$parsedUrl = parse_url($url);
 		$path = isset($parsedUrl['path']) ? $parsedUrl['path'] : '';
-
-		list(,,, $resource, $action) = explode('/', $path);
-		$endpoint = $resource . '/' . $action;
+		if(substr_count($path, '/') === 3)
```

Comment:
there's something I don't like about this solution, but I don't know yet what it is nor how to do it better 😄 this whole method might use some refactoring I guess... but so far so good